README.md

```diff
@@ -24,17 +24,33 @@ The service is subscribed to the following [Keptn CloudEvents](https://github.co
 |       0.6.1      | keptn/dynatrace-service:0.6.2            | 1.13 - 1.15                              |
 |       0.6.1      | keptncontrib/dynatrace-service:0.6.9     | 1.13 - 1.15                              |
 |       0.6.2      | keptncontrib/dynatrace-service:0.7.1     | 1.13 - 1.15                              |
+|       0.7.0      | keptncontrib/dynatrace-service:0.8.0     | 1.13 - 1.15                              |
```

Comment:
FYI: Keptn 0.7 will support Kubernetes 1.14 - 1.18 (even 1.19 maybe), let's reflect that in Dynatrace-Service too!